plugins/list/plugin.js

```diff
@@ -1096,7 +1096,8 @@
 									startItem.getName() == 'li' &&
 									startItem.getParent().getName() in listNodeNames &&
 									!( commonAncestor.getName() in listNodeNames ) &&
-									!( li.contains( startItem ) )
+									!( li.contains( startItem ) ) &&
+									!( li.equals ( startItem ) ) // in Firefox/IE, sublists can incorrectly be detected as being separate lists if not for this
```

Comment:
in IE/Firefox, because the cursor gets moved at a different point than Chrome (somehow) , `li` can be the same element as `startItem`.  This block is only meant for combining lists that aren't part of the same list, but the check fails and incorrectly believes they are separate lists, leading to an infinite loop